core/include/usListenerFunctors.h

```diff
@@ -34,6 +34,25 @@ namespace us {
   class BundleEvent;
 
   typedef std::function<void(const ServiceEvent&)> ServiceListener;
+
+  /**
+   * \ingroup MicroServices
+   *
+   * A \c BundleEvent listener. When a \c BundleEvent is fired, it is
```

Comment:
The behavior when calling bundle listeners is as specified for synchronous listeners in the specs. We do not have async events yet (although they are delivered in a separate thread, they are still synchronized with the bundle operation).